raster-foundry/raster-foundry
# Remove frontend & promote backend

## Overview
This PR does two things:
1. Remove frontend code
2. Move `build.sbt` up to the project directory root to make scala-steward happy

### Checklist

- ~[ ] Swagger specification updated~ (don't think this needs to happen because the API hasn't changed)

### Notes
2cb52ea is a "bonus" commit to avoid adding nexus to the build repo in dev from my onboarding (should this also be removed for CI?)

## Testing Instructions
* Make sure `app-frontend` is gone as is the static asset serving by nginx (`config` and `feature-flags` endpoints consumed by the frontend are still lying around but that will be handled in https://github.com/raster-foundry/raster-foundry/issues/5603)
* Test you can run `sbt` from the project directory root and everything works as normal inside and outside containers
* I'm not sure how to test if `scala-steward` will be happy with this setup but I would be happy to test if someone has an idea of how.

Closes https://github.com/azavea/raster-foundry-platform/issues/1248
